packages/functional-tests/pages/signin.ts

```diff
@@ -39,11 +39,11 @@ export class SigninPage extends BaseLayout {
   }
 
   get continueWithAppleButton() {
-    return this.page.getByRole('button', { name: 'Continue with Apple' });
+    return this.page.getByRole('button', { name: /Continue with Apple/ });
   }
 
   get continueWithGoogleButton() {
-    return this.page.getByRole('button', { name: 'Continue with Google' });
+    return this.page.getByRole('button', { name: /Continue with Google/ });
```

Comment:
Oh?  I don't understand why that's necessary.

Reply:
TLDR; I'll revert this change.

This one was a headscratcher. For a bit, the third party auth tests was actually failing consistently unless I _only_ included `/Google/` and `/Apple/` in the locators 🤔 Couldn't trace the cause (playwright's `pick locator` feature was still recommending the initial locator even though the test was failing) - and reverting back fully is now working again.